Applications/TextEditor/TextEditorWidget.cpp

```diff
@@ -364,12 +364,12 @@ TextEditorWidget::TextEditorWidget()
         m_save_as_action->activate();
     });
 
-    m_line_wrapping_setting_action = GUI::Action::create_checkable("Line wrapping", [&](auto& action) {
-        m_editor->set_line_wrapping_enabled(action.is_checked());
+    m_wrapping_mode_action = GUI::Action::create_checkable("Wrapping Mode", [&](auto& action) {
+        m_editor->set_wrapping_mode_enabled(action.is_checked());
     });
-    m_line_wrapping_setting_action->set_checked(m_editor->is_line_wrapping_enabled());
+    m_wrapping_mode_action->set_checked(m_editor->is_wrapping_mode_enabled());
 
-    auto menubar = GUI::MenuBar::construct();
+   auto menubar = GUI::MenuBar::construct();
```

Comment:
checks are failing because you removed a space here by mistake

Reply:
Thanks for finding that